controllers/base_controller.js

```diff
@@ -151,8 +154,26 @@ module.exports = function BaseControllerModule(pb) {
             onlyThisSite: true,
             siteObj: this.siteObj
         };
+        if(pb.config.localization && pb.config.localization.db){
+            var opts = {
+                where: {_id: self.site}
+            };
+            var queryService = new pb.SiteQueryService({site: self.site, onlyThisSite: true});
+
+            queryService.q("localizations", opts, function (err, result) {
```

Comment:
it appears that all localization overrides for all sites are stored in a single object.  This isn't optimal because we have to load the document that contains items from all sites which we don't need.